config/default.schema.json

```diff
@@ -375,6 +360,28 @@
 			"type": "boolean",
 			"default": false,
 			"description": "Brute force flag during recovery password process"
+		},
+		"PROMETHEUS": {
+			"type": "object",
+			"description": "Prometheus metrics options used by prometheus-api-metrics",
+			"properties": {
+				"METRICS_PATH": {
+					"type": "string",
+					"default": "/metrics",
+					"description": "public metrics route",
+					"pattern": ".*(?<!/)$"
+				},
+				"DURATION_BUCKETS_SECONDS": {
+					"type": "array",
+					"items": {
+						"type": "number",
+						"minimum": 0
+					},
+					"default": [0.01, 0.05, 0.1, 0.2, 0.5, 0.75, 1.0, 3.0, 5.0, 8.0],
```

Comment:
same

Reply:
the seconds mentioned here are separating buckets, means buckets contain all requests that require less or equal the given value.